### ISSUE TITLE
It doesn't work inside nth-child() - pseudo-classes.

### ISSUE THREAD

**Issue by sandrina-p** [shauns/postcss-math#12]
```css
$sum = 2;
$col = 6;

$nChild: resolve($(sum)/$(col)); /* 3 - it works */

.item {
   &:nth-child($(sum)n) { /* 6n - it works */
       margin-right: 0;
   }

   &:nth-child($(nChild)n) { /* resolve(6/2)n - doesn't work, it should be 3n */
       margin-right: 0;
   }

   &:nth-child(resolve($(sum)/$(col))n) { /* resolve(6/2)n - doesn't work, it should be 3n */
       margin-right: 0;
   }
}
```

How can i make math inside pseudo-classes?

**Comment by shauns** [comment]
Hi @sandrina-p thanks for raising -- I've resolved in #13. LMK if this doesn't play well with your use of nested rules, variables etc.

**Comment by sandrina-p** [comment]
Yeh it worked! You're awesome, thanks
